hi-ml-cpath/src/health_cpath/datamodules/base_module.py

```diff
@@ -198,16 +218,11 @@ def _load_dataset(self, tiles_dataset: TilesDataset, stage: ModelKey, shuffle: b
 
         generator = _create_generator(self.seed)
 
-        if stage in [ModelKey.VAL, ModelKey.TEST]:
-            eff_max_bag_size = self.max_bag_size_inf
-        else:
-            eff_max_bag_size = self.max_bag_size
-
         bag_dataset = BagDataset(
             tiles_dataset,  # type: ignore
             bag_ids=tiles_dataset.slide_ids,
-            max_bag_size=eff_max_bag_size,
-            shuffle_samples=shuffle,
+            max_bag_size=self.bag_sizes[stage],
+            shuffle_samples=True,
```

Comment:
Should we explain in the docu that shuffle is always set to True and is there any need to pass shuffle in to this method (`_load_dataset`) anymore?

Reply:
done